### PR TITLE
❗❗❗Guys OMG you have SO much trailing whitespace. Cmon Jeffica.

### PR DESCRIPTION
**Pull request by tmathmeyer**
grep -rIl '[[:blank:]]$' . | xargs sed -i 's/[[:blank:]]*$//'

### CODE REVIEW

**Comment by tmathmeyer** [comment]
@digit01Wave

**Comment by tmathmeyer** [comment]
fixes: https://github.com/PuzzleServer/mainpuzzleserver/issues/807

File: ServerCore/wwwroot/images/ps19/ps19-combined-color.svg
Comment (digit01Wave) [inline ServerCore/wwwroot/images/ps19/ps19-combined-color.svg:1]: Can these images not be modified?

File: ServerCore/wwwroot/lib/bootstrap/dist/fonts/glyphicons-halflings-regular.svg
Comment (digit01Wave) [inline ServerCore/wwwroot/lib/bootstrap/dist/fonts/glyphicons-halflings-regular.svg:1]: Can these images not be modified?

File: ServerCore/wwwroot/images/ps19/ps19-combined-color.svg
Comment (tmathmeyer) [reply ServerCore/wwwroot/images/ps19/ps19-combined-color.svg:1]: The image content isn't modified, just the trailing whitespace within the image.

File: ServerCore/wwwroot/lib/bootstrap/dist/fonts/glyphicons-halflings-regular.svg
Comment (tmathmeyer) [reply ServerCore/wwwroot/lib/bootstrap/dist/fonts/glyphicons-halflings-regular.svg:1]: ditto

**Review by digit01Wave — APPROVED** [review]

**Pull request merged**